_includes/manuals/2.1/1.2_release_notes.md

```diff
@@ -19,17 +19,403 @@ Subnets can be now associated with IPAM plugins providing IP address management
 capabilities via external entities. An experimental implementaion for phpIPAM
 is available as a [Smart Proxy plugin](https://github.com/grizzthedj/smart_proxy_ipam).
 
-### Release Notes
+#### Rails 6 upgrade
+
+Foreman has been updated to run on the latest version of Ruby on Rails at the time of the release, 6.0.2. Many plugins have already been updated as well, but there may be issues with any that have not seen a new release in a while.
+
+#### Support installation on EL8 distributions
+
+Foreman installation is now possible on EL8 distributions. The installer has been updated to support changes needed for these and yum repositories now contain packages for EL8. Note that installation is only supported and tested on RHEL and CentOS, packages may or may not work on Oracle Linux or Scientific Linux.
+
+#### Switch default web server from Passenger to Puma
+
+The default installation of Foreman will now set up Puma behind Apache reverse proxy instead of Passenger with mod_passenger. This will enable us to use Web Sockets in the future as well as additional benefits with regards to certificate handling and unifying development and production setups.
```

Comment:
I'd add info about how to change back to passenger. Note a SELinux boolean will need to be turned on when passenger needs to be used:

    semanage boolean passenger_run_foreman on

Reply:
I'm going to merge this now as is so it's available for the RC1 announcement, please open a follow-up PR with what needs to be done.